src/custom/brandcolors/brandcolors_custom.less

```diff
@@ -0,0 +1,17 @@
+
+// https://itch.io/press-kit/#color-palette
+@bc-itch-io-1: #fa5c5c;
+@bc-itch-io-2: #ff8080;
+
+// https://github.com/gamejolt/frontend-lib/blob/master/stylus/common/_variables.styl#L4
+@bc-gamejolt-1: #cf0;
+@bc-gamejolt-2: #2f7f6f;
+@bc-gamejolt-3: #ff3fac;
+@bc-gamejolt-4: #31d6ff;
+
+// https://hitbox.desk.com/customer/en/portal/articles/1434405-can-i-use-the-hitbox-logo-on-my-stream-overlay-website-etc-?b_id=16071
+@bc-hitbox-tv-1: #9C0;
+
+// Pulled from there logo
+@bc-one-hour-game-jam-1: #ff0066;
+@bc-one-hour-game-jam-2: #4A3457;
```

Comment:
This also should have been caught by the linter. Newline at end of file please.

Reply:
fixed